scripts/build-desktop.sh

```diff
@@ -306,16 +303,11 @@ function bundleLinux() {
 
 if is_macos; then
   function getQtFullOutPathFromNixStore() {
-    local qtFullDerivationPath=$(nix show-derivation -f $STATUSREACTPATH/default.nix | jq -r '.[] | .inputDrvs | 'keys' | .[]' | grep qt-full)
-
-    echo $(nix show-derivation $qtFullDerivationPath | jq -r '.[] | .outputs.out.path')
+    echo $QT_PATH
   }
 
   function getQtBaseBinPathFromNixStore() {
-    local qtFullDerivationPath=$(nix show-derivation -f $STATUSREACTPATH/default.nix | jq -r '.[] | .inputDrvs | 'keys' | .[]' | grep qt-full)
-    local qtBaseDerivationPath=$(nix show-derivation $qtFullDerivationPath | jq -r '.[] | .inputDrvs | 'keys' | .[]' | grep qtbase)
-
-    echo $(nix show-derivation $qtBaseDerivationPath | jq -r '.[] | .outputs.bin.path')
+    echo $QT_BASEBIN_PATH
```

Comment:
If we are just changing them to variables, shouldn't we just get rid of methods like `getQtBaseBinPathFromNixStore` or `getQtFullOutPathFromNixStore` and use the variables directly?

Reply:
I was leaving the methods so the name would act as a comment, but I guess we don't really need it. I'll remove them.